app/index.html

```diff
@@ -8,156 +8,207 @@
-        </section>
-        
-        <section>
-            <ul class="social-icons">
-                <li><a href="#"><img src="./assets/social-icons/twitter.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/Linkedin_icon.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/medium.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/github.png" alt=""></a></li>
-                <li><a href="#"><img src="./assets/social-icons/angellist.png" alt=""></a></li>
-            </ul>
-        </section>
-
-        <a href="" class="scroll-down-btn"></a>
-           
-    </main>
+    <div id="container">
+        <header class="primary-font primary-text-color">
+            <div class="center align-left" id="logo">
+                <a href="#" id="logo-text" class="primary-text-color">Welcome</a>
+            </div>
+            <nav class="center align-right">
+                <img id="menu" src="assets/menu_button.png" alt="">
+                <ul id="nav-elements">
+                    <li><a href="#works" class="nav-link">Portfolio</a></li>
+                    <li><a href="#about" class="nav-link">About</a></li>
+                    <li><a href="#form-container" class="nav-link">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main id="content">
+            <section>
+                <h1 class="center-text primary-text-color primary-font heading u-margin">Hey there. I&apos;m Mariane <br><span class="secondary-text-color">I&apos;m a software developer</span> </h1>
+    
+                <p class="paragraph primary-text-color"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don&apos;t hestiate to contact me.</p>
+            </section>
+            
+            <section>
+                <ul class="social-icons">
+                    <li><a href="#"><img src="./assets/social-icons/twitter.png" alt=""></a></li>
+                    <li><a href="#"><img src="./assets/social-icons/Linkedin_icon.png" alt=""></a></li>
+                    <li><a href="#"><img src="./assets/social-icons/medium.png" alt=""></a></li>
+                    <li><a href="#"><img src="./assets/social-icons/github.png" alt=""></a></li>
+                    <li><a href="#"><img src="./assets/social-icons/angellist.png" alt=""></a></li>
+                </ul>
+            </section>
+    
+            <a href="" class="scroll-down-btn"></a>
+               
+        </main>
+    </div>
+    
 
     <section id="works" class="u-margin">
         <h2 class="primary-text-color primary-font heading">My Recent works</h2>
```

Comment:
#### OPTIONAL  (Issue occurring between 992 and 1100 pixels)

I'm going to mark this as a suggestion rather than a requirement because the last reviewer didn't mention it so you can apply it if you wish (Highly recommend fixing this issue so that your portfolio looks good on all sizes) 🧐  

There are some width issues with your works and your tech sections that is causing your website to look like this..

#### Works section
![Captura](https://user-images.githubusercontent.com/15807343/149412431-78088208-1f8e-4279-b645-e243e787129b.PNG)

#### About me section 
![Captura2](https://user-images.githubusercontent.com/15807343/149412566-f64942b8-5068-48c3-a9fb-a5d702653955.PNG)

As you can see, the project cards are oversizing the width of the screen in the first image and in the case of your languages/skills/frameworks, the cards are too small.